01-shapefile-attributes.Rmd

```diff
@@ -36,33 +38,35 @@ values.
 ###Goals / Objectives
 After completing this activity, you will:
 
- * Be able to query shapefile attributes
- * Be able to subset shapefiles based on specific attributes
+ * Be able to query shapefile attributes.
+ * Be able to subset shapefiles based on specific attributes.
+ * Know how to plot shapefiles with objects differentiated by attribute.
```

Comment:
? 
* Know how to plot a shapefile, colored by unique attribute values.

the bullet below seems redundant with this one?